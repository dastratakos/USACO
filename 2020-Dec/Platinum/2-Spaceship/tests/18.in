60 60 60
111111111111101011011111100011100111111100101110111111011111
010111011101101111011111111101001110101100111011101010111011
111011000110101111111111111110001000011111111100011111011101
010011011101110010011000011100101010010111110111010101110101
110111000111111111111011010111101110101111111111001111010111
000101000000100100110010001111001101110000100011110111011110
101010110111000110000110001011101101111111111111010011011011
011011010011110001110011110110011011101010011010011111111111
111111110110111101111111011111101111111000111100111100001010
100101011101010011100101101010111110101101101111110011101111
111011111011001010000101010111110111101111111111110011101101
101111111110010011011111100111001101111100010100110100101111
110111101100010101111101111101111011011101011101101001010110
111110111100101111111110011110110111111111010100011101101000
111111111101110111000001110101001011101111110100100101110100
001101111111000111111011101111010100111110011011111011011101
110001110100001011011010010000111100001001010100101101011110
111111111110011111110010011110011010110101100011111111011111
101001010111100011111100011101111100111101111011111100100100
101011011111111111111011010111110100010000111101010100100111
100101100001001111100110001011101111111111001111001111111101
111101111111001011101101111111011111111101011011111011111101
111101111001100101100011111100010011111011110100101101111111
101010111111101011110111111101101111001111111111111111111111
011111111111000100011111101111001100111011101101111110111100
111011011101011011011111110000100111000101110010011101100001
010111110111011111111101000101111100111111111101010111101101
111111011010011011000000100110110011011110011001101011111111
011111000101010111110111100111110010100011111001111001111010
101000101111111111111010110111011111100001010101111011101100
011100000110111010010100110011111110111111000111010101111111
001101100111011110011111110110011100110111111111111101111001
100100111110101101111111101011111111010011011010111110101010
101101011111011011011110111001011111100110101110111110100111
001011100101010111110110111010111011111110111100111111111111
110100011101011001111111011000101001110000111111100000101010
010101001100010100111101011000101011100011011010110110100111
111111000100111111010110110001011010011011111010101110110111
111110010111011011001010111110001001101011100111011101011010
111101010101111100000010000111011100111000111111001011110011
111111011101011011100111111101011001110011110011110011010101
111101010011000111111010111110110111101111110011011101001101
011000110101111111011111001001001100011110111011011110111110
111011011000101101001111110001011111100101001110111110010111
111101001110111111001100011101101101100110001111111111101111
111101100110110010101101111111111111100011011111111110110100
101110010100111101111101111010111111111101111111110001011011
101011101011011001111111101110111001100111010100111111101110
000101001101010110011011001110011110111110010100001110100101
010010011111001101100111111101000101101001001111011100100111
111111111101111010110011110110111001001111110000111111100111
011111111101101111101111011111101001100111111111111111010111
111101001111110111101101010010101111111011111101110111110111
101001010111011011101110100110110001011111011110101111101011
111001101101111111010101010101110001101111111011100001111100
010100010111110001011101111110101001010110011101111110111011
100011111100101010011010111001010100110111101111001100101011
010111110111011110111110110000110111111010011010110111111011
111101111100000010101100110100010101010101111110001001101111
011100110111111000111100001110011010011010011011100111100000
3 35 1 7
4 26 4 12
3 29 4 23
33 60 1 46
5 21 3 53
4 43 50 8
2 25 42 19
3 49 3 17
34 39 5 15
2 21 3 16
31 3 3 50
1 16 1 5
2 1 41 26
56 48 4 33
4 16 49 28
50 26 4 29
1 37 3 11
5 46 2 13
1 52 5 45
26 3 2 58
5 1 1 45
1 43 2 52
4 23 5 9
13 30 5 20
3 60 47 55
5 21 35 13
3 41 35 42
3 17 51 15
1 23 25 7
3 51 8 53
17 50 5 10
4 24 35 54
4 40 4 42
2 38 1 2
1 10 18 14
5 56 3 43
2 25 1 54
4 60 2 46
3 55 3 15
4 15 2 2
3 22 5 47
3 60 1 54
5 22 4 52
4 47 2 44
40 36 2 8
2 40 4 16
3 27 5 28
11 1 25 17
5 2 5 34
21 41 5 27
1 7 5 8
3 56 4 4
4 2 4 39
52 14 41 52
5 5 3 38
4 32 40 22
22 29 18 27
4 11 4 42
23 14 4 29
4 11 4 11
