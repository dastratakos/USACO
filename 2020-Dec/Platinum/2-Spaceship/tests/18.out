259028760
490900362
1475599
40218854
211061496
173866277
532408525
876942468
509608423
328407584
151741425
801434871
623918660
591986341
991361051
832240383
521687025
419830876
360695386
861129876
316170930
782916111
569838182
21981289
627393089
572542735
843137642
94861269
998736688
451955215
918516269
83217109
899780175
132593492
332505794
327577698
189574063
320927593
99997899
760037631
835957450
234410844
261192842
352731142
378045196
239734841
574890689
783888576
107996581
620515090
292609734
321771636
337697633
704256799
291160076
649631434
558434321
521502797
669004599
491825718
